src/Fetcher/FetcherInterface.php

```diff
@@ -11,10 +11,16 @@
 
 namespace App\Fetcher;
 
+use App\Entity\Conference;
 use Symfony\Component\Form\FormBuilderInterface;
 
 interface FetcherInterface
 {
+    /**
+     * @param array<mixed> $configuration
+     *
+     * @return \Generator<Conference>\null
```

Comment:
Same error!

I usually don't repeat the same comment on each error because

1. It's boring for me and you
2. It takes too much place in the UI
3. I expect you to learn, and try to enhance your code according to your new learning, and not fixing all my comments without thinking. The goal is not to be the fastest, but the smarter :)

so from now on, please fix all occurrence of an error :)